### PR TITLE
Further RU update

### PR DESCRIPTION
**Pull request by paladinkolovrat**

### CODE REVIEW

File: application/src/main/resources/locale/blueprint/horizons/names.csv [hunk 1]
```diff
@@ -98,4 +98,4 @@ blueprint.horizons.name.thrusters,Thrusters,Thrusters,Triebwerke,Propulsores,Pro
 blueprint.horizons.name.torpedo_pylon,Torpedo Pylon,Torpedo Pylon,Torpedo Pylon,Pilón de torpedo,Pylône de torpille,Pilão de torpedo,Торпедная установка
 blueprint.horizons.name.caustic_sinks,Caustic Sinks,Caustic Sinks,Ätzbehälter,,,,Антикоррозийные отводы
 blueprint.horizons.name.fsd_injection,FSD Injection,FSD Injection,FSA-Einspeisung,Inyección de FSD,Injection de FSD,Injeção de FSD,FSD инъекция
-blueprint.horizons.name.caustic_sink_launcher,Caustic Sink Launcher,Caustic Sink Launcher,Ätzbehälterwerfer,,,,
\ No newline at end of file
+blueprint.horizons.name.caustic_sink_launcher,Caustic Sink Launcher,Caustic Sink Launcher,Ätzbehälterwerfer,,,,Антикоррозийная катапульста
```
Comment (cptn-solo) [inline application/src/main/resources/locale/blueprint/horizons/names.csv:101]: Очепятка

**Review by cptn-solo — APPROVED** [review]
We can merge it, just apply my RP on top to fix spelling

File: application/src/main/resources/locale/blueprint/horizons/names.csv (hunk 1 above)
Comment (paladinkolovrat) [reply application/src/main/resources/locale/blueprint/horizons/names.csv]: Oopsie

**Comment by jixxed** [comment]
Merged #170 instead

**Pull request closed without merging**